src/zenml/enums.py

```diff
@@ -114,3 +114,9 @@ class SecretValidationLevel(StrEnum):
     SECRET_AND_KEY_EXISTS = "SECRET_AND_KEY_EXISTS"
     SECRET_EXISTS = "SECRET_EXISTS"
     NONE = "NONE"
+
+
+class IdentifySource(StrEnum):
```

Comment:
IMO the name could be a little more expressive here, something with analytics in there? `AnalyticsEmailSource`/`AnalyticsEventSource`?

Reply:
Docstring too ;)